config/index.js

```diff
@@ -287,6 +287,7 @@ module.exports = {
   },
   FEATURE_FLAGS: {
     GOT: /true/i.test(process.env.FEATURE_FLAG_GOT),
+    BANNER_VISIBLE: /true/i.test(process.env.FEATURE_FLAG_BANNER_VISIBLE),
```

Comment:
```suggestion
    WHATS_NEW_BANNER: /true/i.test(process.env.FEATURE_FLAG_WHATS_NEW_BANNER),
```

I think this is a clearer name, but I don't really mind since we'll get rid of this feature flag when we release!